src/components/Icon/Icon.tsx

```diff
@@ -32,8 +32,4 @@ export const Icon = ({
     );
 };
 
-// When used with uikit components Icon is considered as text element and corresponding styles are applied
-// IconWrapper overrides displayName to 'Icon', so it will be considered as an icon with right styles
-export const IconWrapper = (props: IconProps) => <Icon {...props} />;
-
-IconWrapper.displayName = 'Icon';
+Icon.displayName = 'Icon';
```

Comment:
When update to new uikit, some problems with `Icon` with overrided styles occurred. So I updated `Icon` display name and removed almost alll `yc-button__text` style overrides